pkg/build/norelease.go

```diff
@@ -0,0 +1,7 @@
+//go:build !release
+
+package build
+
+func initReleaseOptions() {
+	CheckChildAlreadyExists = true
+}
```

Comment:
I think you can skip option.go and have this in norelease.go
```
const CheckChildAlreadyExists = true
```
and this in release.go
```
const CheckChildAlreadyExists = false
```